### PR TITLE
support stable versions

### PR DESCRIPTION
**Pull request by GiladShoham**

### CODE REVIEW

File: teambit.bvm/list/version-list/remote-version-list.ts [hunk 1]
```diff
@@ -1,13 +1,52 @@
...
+import { VersionList } from './version-list';
 
 export class RemoteVersionList extends VersionList {
-  constructor(public entries: RemoteVersion[]){
+  constructor(public entries: RemoteVersion[]) {
     super(entries);
-  };
+  }
+
+  toTable() {
```
Comment (GiladShoham) [inline teambit.bvm/list/version-list/remote-version-list.ts:10]: this should be in the command layer

File: teambit.bvm/commands/list/list.ts [hunk 2]
```diff
@@ -1,43 +1,47 @@
...
+          alias: ['r'],
+          describe: 'show versions exist in the remote server',
+          default: false,
+          type: 'boolean',
+        },
+        'release-type': {
+          alias: ['t'],
+          describe: 'show only the stable versions',
+          default: 'stable',
+          type: 'string',
```
Comment (GiladShoham) [inline teambit.bvm/commands/list/list.ts:21]: make it choices  - https://github.com/yargs/yargs/blob/master/docs/api.md#choiceskey-choices

**Comment by GiladShoham** [comment]
Add option for global config for release type

File: teambit.bvm/list/version-list/remote-version-list.ts (hunk 1 above)
Comment (guysaar223) [reply teambit.bvm/list/version-list/remote-version-list.ts]: fixed

File: teambit.bvm/commands/list/list.ts (hunk 2 above)
Comment (guysaar223) [reply teambit.bvm/commands/list/list.ts:52]: fixed

**Pull request merged**